core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -235,6 +251,13 @@ class TradeEvaluation {
             }
         }
     }
+    fun distanceDiscount(civInfo: CivilizationInfo, capitalcity: CityInfo, city: CityInfo): Int{
```

Comment:
discount of what? Are we buying or selling? Who do the cities belong to? There's a lot of renaming to be done here to make this immediately readable

Reply:
The Cities can belong to us or the AI. This is evaluating sell cost

Reply:
renamed